core/chains/evm/txmgr/evm_tx_store.go

```diff
@@ -1152,74 +1072,47 @@ SELECT * FROM evm.txes WHERE from_address = $1 AND nonce = $2 AND state IN ('con
 	return
 }
 
-func updateEthTxAttemptUnbroadcast(ctx context.Context, orm *evmTxStore, attempt TxAttempt) error {
-	if attempt.State != txmgrtypes.TxAttemptBroadcast {
-		return errors.New("expected eth_tx_attempt to be broadcast")
-	}
-	_, err := orm.q.ExecContext(ctx, `UPDATE evm.tx_attempts SET broadcast_before_block_num = NULL, state = 'in_progress' WHERE id = $1`, attempt.ID)
-	return pkgerrors.Wrap(err, "updateEthTxAttemptUnbroadcast failed")
+func updateEthTxAttemptUnbroadcast(ctx context.Context, orm *evmTxStore, attemptIDs []int64) error {
+	_, err := orm.q.ExecContext(ctx, `UPDATE evm.tx_attempts SET broadcast_before_block_num = NULL, state = 'in_progress' WHERE id = ANY($1)`, pq.Array(attemptIDs))
+	return err
 }
 
-func updateEthTxUnconfirm(ctx context.Context, orm *evmTxStore, etx Tx) error {
-	if etx.State != txmgr.TxConfirmed {
-		return errors.New("expected tx state to be confirmed")
-	}
-	_, err := orm.q.ExecContext(ctx, `UPDATE evm.txes SET state = 'unconfirmed' WHERE id = $1`, etx.ID)
-	return pkgerrors.Wrap(err, "updateEthTxUnconfirm failed")
+func updateEthTxUnconfirm(ctx context.Context, orm *evmTxStore, etxIDs []int64) error {
+	_, err := orm.q.ExecContext(ctx, `UPDATE evm.txes SET state = 'unconfirmed', error = NULL WHERE id = ANY($1)`, pq.Array(etxIDs))
+	return err
 }
 
-func deleteEthReceipts(ctx context.Context, orm *evmTxStore, etxID int64) (err error) {
+func deleteEthReceipts(ctx context.Context, orm *evmTxStore, etxIDs []int64) (err error) {
 	_, err = orm.q.ExecContext(ctx, `
 DELETE FROM evm.receipts
 USING evm.tx_attempts
 WHERE evm.receipts.tx_hash = evm.tx_attempts.hash
```

Comment:
Probably out of scope but this WHERE seems irrelevant

Reply:
Hmmm seems like it but I'd rather keep the query as is since it's carrying over from the previous code in case I'm misreading it.